spock-spring/src/test/groovy/org/spockframework/spring/docs/DetachedJavaConfig.groovy

```diff
@@ -0,0 +1,25 @@
+package org.spockframework.spring.docs
+
+import org.springframework.context.annotation.Bean
+import spock.mock.DetachedMockFactory
+
+//tag::javaconfig[]
+public class DetachedJavaConfig {
+  def mockFactory = new DetachedMockFactory()
+
+  @Bean
+  GreeterService serviceMock() {
+    return mockFactory.Mock(GreeterService)
+  }
```

Comment:
Yes it would. I guess it's just a different pattern that doesn't require the `DetachedMockFactory` instance. It does exactly the same thing.